src/main/java/com/karankumar/bookproject/backend/statistics/PageStatistics.java

```diff
@@ -64,7 +64,7 @@ public Integer calculateAveragePageLength() {
                                                    .sum();
         int booksWithPagesSpecified = booksWithPageCount.size();
         if (booksWithPagesSpecified == 0) {
-            return null;
+            return Optional.ofNullable(null);
```

Comment:
The better option here could be to change the method signature by returning `Optional<Integer>` and then return `Optional.empty()`